docs/tempo/website/configuration/_index.md

```diff
@@ -569,6 +569,13 @@ storage:
             # The maximum number of requests to execute when hedging. Requires hedge_requests_at to be set.
             [hedge_requests_up_to: <int>]
 
+           # Optional
+           # Example: "block_tags: {'key': 'value'}"
+           # A map key value string for user tags to store on the S3 block objects. This string helps set up filters in S3 lifecycles.
+           # The tags are not set on index.json.gz files since these files are rewritten often.
```

Comment:
this doesn't match the behavior anymore. it's weird to me to make a "tags" config option that applies to all objects except one. is there a better way to do this?

also, why "block_tags" and not "user_tags"?

Reply:
I agree, but then the complexity of the config grows.  We set defaults and let a map decide the options.

Perhaps `object_tags`?

Reply:
i feel like any defaults other than empty would be strange.